Parser/pegen/pegen.c

```diff
@@ -310,6 +310,10 @@ get_error_line(char *buffer, int is_file)
         newline = strchr(buffer, '\n');
     }
 
+    while (is_file && newline > buffer && newline[-1] == '\n') {
+        --newline;
+    }
```

Comment:
Ugh. Why test `is_file` each time through the loop?

Reply:
Whops, I was doing some testing with the full condition and forgot to re-add the outer `if`. Sorry, is getting late here :sweat_smile: 

Fixed it in 97d320a.